### ISSUE TITLE
Exception when trying to open list of objects having multiple pages.

### ISSUE THREAD

**Issue by Waszker** [MaistrenkoAnton/django-material-admin#139]
Hi, first of all thank you very much for this awesome project!

Today I tried updating the package to the recently released version supporting Django 3.2.
I have noticed that when trying to open admin views containing list of objects, if there are a lot of those items (in previous project versions such pages displayed page number like "1, 2, 3, ..., 997, 998", with the "..." (three dots) in the middle) the exception is raised:

```
[...]
project_1                |   File "/usr/local/lib/python3.9/site-packages/material/templatetags/material.py", line 109, in material_paginator_number
project_1                |     i + 1,
project_1                |   File "/usr/local/lib/python3.9/site-packages/django/utils/functional.py", line 180, in __add__
project_1                |     return self.__cast() + other
project_1                | TypeError: can only concatenate str (not "int") to str
project_1                | "GET /admin/core/items/ HTTP/1.1" 500 475520
```

Let me know if I can help any further!

**Comment by Waszker** [comment]
One additional note - in the views where there are not so many pages, the numbering starts from "2" instead of "1".
Maybe this information will help to pinpoint the bug.

**Comment by MaistrenkoAnton** [comment]
Thanks,
fixed in `v1.8.4`

**Comment by Waszker** [comment]
Works like a charm.
Thank you very much for such fast response and fix!
